functions/add_user/__init__.py

```diff
@@ -0,0 +1,44 @@
+from shared_db import FAPDatabase
+import azure.functions as func
+import json
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    with FAPDatabase('benutzer') as benutzer_collection:
+        body = req.get_json()
+
+        if benutzer_collection.find_one({
+            'loginName': body['loginName']
+        }):
+            return func.HttpResponse(
+                body=json.dumps({
+                    'ergebnis': False,
+                    'meldung': 'Benutzername bereits vorhanden.'
+                }),
+                status_code=200,
+                mimetype='applicatoin/json',
+                charset='utf-8'
+            )
+
+        benutzer_collection.insert_one({
+            'loginName': body['loginName'],

```

Comment:
Unique index auf loginName sonst Dopplung möglich.
Race Condition aber mit Performance Anforderungen des Projektes optional.

Reply:
OK, I will wait for your commit. <3